asplib_modules/FFT/CMakeLists.txt

```diff
@@ -21,15 +14,17 @@ file (GLOB_RECURSE ASPLIB_FFT_HEADERS
 
 asplib_source_group("${ASPLIB_FFT_SOURCES}")
 asplib_source_group("${ASPLIB_FFT_HEADERS}")
+
 add_library(asplib_FFT STATIC ${ASPLIB_FFT_SOURCES} ${ASPLIB_FFT_HEADERS})
+add_library(asplib::FFT ALIAS asplib_FFT)
+target_link_libraries(asplib_FFT PRIVATE asplib::Core KissFFT::KissFFT)
```

Comment:
PRIVATE, PUBLIC, INTERFACE is best explained here:
https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html
Basically: 
- PRIVATE = use it to compiling/linking this library
- INTERFACE = propagate it to users of the library, but don't use it for this library
- PUBLIC = both
(for target_include_directories: a rule of thumb is: if you use the dependecy only in your cpp files or private headers -> PRIVATE. If you use it in public headers -> INTERFACE. If both: PUBLIC)